Main.py

```diff
@@ -0,0 +1,42 @@
+from functions import *
```

Comment:
Asterisk import is not appreciated in production code as it makes the import explicit and you never know what's imported/available without navigating to the imported module. Things may get even more complicated if the inside the imported modules the available objects are collected dynamically.

In other words, avoid using asterisk imports unless you are 100% sure you need it.